cmd/root.go

```diff
@@ -105,8 +105,9 @@ func initConfig() {
 		}
 
 		// Search config in home directory with name "mbmd" (without extension).
-		viper.AddConfigPath(".")    // optionally look for config in the working directory
-		viper.AddConfigPath("/etc") // path to look for the config file in
+		viper.AddConfigPath(".")     // optionally look for config in the working directory
+		viper.AddConfigPath("/etc")  // path to look for the config file in
+		viper.AddConfigPath("/perm") // path to look for the config file in
```

Comment:
Der war vorhin doch noch nicht drin- wäre schön das in die `gokrazy.go` auszulagern, order warum hast Du es jetzt doppelt?

Reply:
Huch!? Stimmt, das war nicht indentiert. Da ist mir wohl beim Rebase was kaputtgegangen. Sorry fuer die Verwirrung.